vue-frontend/src/components/portfolio/ElementSection.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <section class="tw-bg-white tw-relative">
```

Comment:
Haven't we implemented parallax in this section?

Reply:
No we do't used parallax in this div